ui/components/app/collectible-details/collectible-details.js

```diff
@@ -236,23 +237,28 @@ export default function CollectibleDetails({ collectible }) {
               {t('source')}
             </Typography>
             <Typography
-              color={COLORS.PRIMARY_DEFAULT}
+              color={COLORS.TEXT_DEFAULT}
               variant={TYPOGRAPHY.H6}
               boxProps={{
                 margin: 0,
                 marginBottom: 4,
               }}
-              className="collectible-details__image-link"
+              className="collectible-details__image-source"
             >
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={collectibleImageURL}
-                title={collectibleImageURL}
-              >
-                {collectibleImageURL}
-              </a>
+              {collectibleImageURL}
             </Typography>
+            <button
+              className="collectible-details__contract-copy-button"
+              onClick={() => {
+                handleSourceCopy(collectibleImageURL);
+              }}
+            >
+              {sourceCopied ? (
+                t('copiedExclamation')
+              ) : (
+                <Copy size={15} color="var(--color-icon-alternative)" />
+              )}
+            </button>
```

Comment:
@georgewrmarshall / @garrettbear  / @NidhiKJha  do we have a "Icon button" component in the design system?

Reply:
Yes, docs here https://metamask.github.io/metamask-storybook/?path=/docs/ui-components-component-library-button-icon-button-icon-stories-js--default-story#buttonicon

Reply:
@ryanml do you plan to use this button instead?

Reply:
@adonesky1 I've updated this to use the `<ButtonIcon/>` component. Screenshots in the PR description have been updated, thanks!